mergexml.js

```diff
@@ -63,15 +57,137 @@
     var XML_PI_NODE = 7;
     var that = this;
 
-    that.Init = function () {
+    var Init = function () {
+      that.error = {};
+      /* detect NodeJS environment */
+      if (typeof process !== 'undefined' && process.versions && process.versions.node) {
+        var p = typeof opts === 'object' && typeof opts.path === 'string' ? opts.path : '';
+        try {  //obtain xml support
+          global.XPathEvaluator = require(p + 'xpath');
+          global.DOMParser = require(p + 'xmldom').DOMParser;
+          global.XMLSerializer = require(p + 'xmldom').XMLSerializer;
+        } catch (e) {
```

Comment:
These identifiers are used because the window object has the same ones. That allows to operate with common names after the initialization. No matter, window or global object exists.

Reply:
Ah yes, I see your point. Thanks. And since as you found out, to my big surprise, that browserify does **not choke** on require()'s that include variables (it skips them), my objection is not valid.